src/abaqus/UtilityAndView/SymbolicConstant.py

```diff
@@ -21,7 +24,7 @@ class SymbolicConstant:
             from abaqusConstants import *
     """
 
-    def __init__(self, text: str):
+    def __init__(self, name: str, scdId:int = -1) -> None:
```

Comment:
The first argument name should be `text` according to the [abaqus documentation](https://help.3ds.com/2022/English/DSSIMULIA_Established/SIMACAEKERRefMap/simaker-c-utlsymbolicconstantpyc.htm?contextscope=all).
![image](https://user-images.githubusercontent.com/47051427/184583157-54baf248-0d5c-4ccc-a1e8-66393550f0cc.png)


What does the argument `scdId` represent?

Reply:
Aha, I know the `scdId` is from the `abaqusConstants`:
```python
class SymbolicConstantType(str):
"""
SymbolicConstant(name <,scdId=-1>)
    Abaqus/CAE SymbolicConstant implemented in Python.
    SymbolicConstant('ON'|'OFF') will return an AbaqusBoolean instance.
"""
```

Reply:
> SymbolicConstant('ON'|'OFF') will return an AbaqusBoolean instance

I think we should implement this.

Reply:
> The first argument name should be `text` instead of `name` according to the abaqus documentation
> What does the argument `scdId` represent?
> Aha, I know the `scdId` is from the `abaqusConstants`, but I think it is not necessary:  

Yeah, that came from inspection of the `SymbolicConstant` class.

```python
# Python 2.7.15 for Abaqus 2021
>>> from abaqusConstants import *
>>> help(SymbolicConstant.__init__)
Help on method __init__ in module symbolicConstants:

__init__(self, name, scdId=-1) unbound symbolicConstants.SymbolicConstant method
```

But you're right, it is really not necessary for this purpose. I changed it to match the documentation.

Reply:
> > SymbolicConstant('ON'|'OFF') will return an AbaqusBoolean instance
> 
> I think we should implement this.

Done in [07947a3](https://github.com/haiiliin/abqpy/pull/468/commits/07947a353304de27b22b5fedaf373aa5aac508f6)